web/src/containers/BudgetSummary.js

```diff
@@ -42,16 +45,18 @@ DataRow.propTypes = {
 
 const DataRowGroup = ({ data, entries, year }) => (
   <Fragment>
-    {Object.keys(data).map(key => (
-      <DataRow
-        key={key}
-        category={key}
-        data={data[key][year]}
-        entries={entries}
-        title={categoryLookup[key]}
-        year={year}
-      />
-    ))}
+    {Object.keys(data)
+      .sort(categorySort)
+      .map(key => (
```

Comment:
Would it make sense to do the same thing here as my comment in #2293, where the `categoryLookup` is pre-sorted so it doesn't have to be done on each instance of the `DataRowGroup`?

E.g.,

```js
const categories = Object.keys(categoryLookup).sort(categorySort);
```

And then here it'd be:

```js
categories.map(key => (...
```

Reply:
yes, agreed!